website/docs/developer/development-guide/code/versioning-releases.md

```diff
@@ -207,6 +207,9 @@ After hitting the play button on `Deploy Cumulus buckets and Cumulus and Orca mo
 - EARTHDATA_CLIENT_ID
 - EARTHDATA_CLIENT_PASSWORD
 - CUMULUS_ORCA_DEPLOY_TEMPLATE_VERSION
+- ORCA_RECOVERY_BUCKET_NAME

```

Comment:
Let's add a little description for this variable. Something like "The name of the bucket outside of ORCA that files will be recovered to" and give the name used in your bucket automation.

Reply:
Added.